### ISSUE TITLE
Accessibility: use descriptive, action-oriented wording for in-line links

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#665]
#### Describe the task

Have descriptive link text for inline hyperlinks to assist screen readers in conveying a clear purpose and destination for the link. Right now links are not always descriptive and can lack context.

#### Acceptance Criteria

- [ ] **Successful new registration submission screen:**

 
**Change:** 
To report any of these events, please use the following link: [Link to form](https://submit.digital.gov.bc.ca/app/form/submit?f=d26fb011-2846-44ed-9f5c-26e2756a758f)
Staff will review the information provided and administer changes in BCIERS as needed. For questions reach out to [GHGRegulator@gov.bc.ca](mailto:GHGRegulator@gov.bc.ca).
 
TO
 
“[Report an event](url) and staff will review the information provided and administer changes in BCIERS as needed. For questions reach out to ghgregulator@gov.bc.ca.”

- [ ] **Submit a report**
1st screen - Review operation information

**Change**
The information shown on this page is data entered in Administration. You can edit Operation Information here but it will only apply to this report. To apply Operation information edits to all of your reports, please edit this information in [Administration.](https://cas-bciers-frontend-test.apps.silver.devops.gov.bc.ca/administration)

TO 

"Any edits to operation information made here will only apply to this report. You can [update operation information](Operation information page) in the operations page."

- [ ] **Submit a report**

2nd screen - Person responsible
 
**Change**
See that information is incorrect or need to add a new contact? Update it here in [Contacts.](https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/contacts)

TO
 
[Add or edit contact information](url).
 

- [ ] **Submit a report**

3rd screen - Review facilities [Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=36177-7310&t=LLq0QBuKYhQSKzws-4)

**Change**
Linear Facilities Operations must register and report for all large and medium facilities, as well as a small aggregate, if applicable. Don’t see a facility? [Add it here](https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations/954c0382-ff61-4e87-a8a0-873586534b54/facilities/add-facility) and then click on the ‘Sync latest data from Administration’ button to update this list of facilities.
 
TO
 
Missing a facility? [Add a new facility](url) and sync latest changes to update this facility list. (line break) Linear Facilities Operations must register and report for all large and medium facilities, as well as a small aggregate, if applicable.
